com.unity.formats.alembic/Editor/Importer/AlembicImporter.cs

```diff
@@ -20,6 +19,22 @@
 [assembly: InternalsVisibleTo("Unity.Formats.Alembic.UnitTests.Editor")]
 namespace UnityEditor.Formats.Alembic.Importer
 {
+    class AlembicAssetPostProcessor : AssetPostprocessor
+    {
+#if UNITY_2021_3_OR_NEWER
+        static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths, bool didDomainReload)
+#else
+        static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
+#endif
```

Comment:
Are we really supporting version older than 2021.3??

Technically, Alembic 2.3.X is only released on 2022.3+. Should we simply update the minimum version allowed and don't bother with `#if/#else`?

Reply:
It is supporting it, but it's not released in it. So by default, when a user open Unity 2020.3, it's not the version they'll have.
But yes, I forgot that bumping the Unity version requires a new major. So I guess we have to stick to #if/#else for now 😞